glpk.d.ts

```diff
@@ -18,7 +18,7 @@ interface LP {
     }[],
     binaries?: string[],
     generals?: string[],
-    settings?: { mipGap?: number, tmLim?: number, msgLev?: number }
+    options?: { mipGap?: number, tmLim?: number, msgLev?: number, preSolve?: number }
```

Comment:
for this comment, I think we need to declare a new interface, like:

```
interface Options {

}
```
and then reference this interface here and below on the solve.